### PR TITLE
Ignore packages when the path contains "thirdparty"

### PR DESCRIPTION
**Pull request by eggerk**

### CODE REVIEW

File: ignored_package_thirdparty/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ignored_test_thirdparty)
+
+find_package(catkin_simple REQUIRED)
+catkin_simple()
```
Comment (HannesSommer) [inline ignored_package_thirdparty/CMakeLists.txt]: Could you add `ALL_DEPS_REQUIRED` here? (see email)

Comment (HannesSommer) [reply ignored_package_thirdparty/CMakeLists.txt]: And maybe also for the other ignored packages?

**Comment by schneith** [comment]
Wooow. That was fast.

Thanks for the implementation.

**Comment by HannesSommer** [comment]
What is not tested here is of course that it would still build the packages not ignored packages depend on. But I'm pretty optimistic.

**Pull request merged**